cmd/tailscale/cli/exitnode.go

```diff
@@ -173,11 +173,13 @@ func hasAnyExitNodeSuggestions(peers []*ipnstate.PeerStatus) bool {
 // a peer. If there is no notable state, a - is returned.
 func peerStatus(peer *ipnstate.PeerStatus) string {
 	if !peer.Active {
+		lastseen := lastSeenFmt(peer.LastSeen)
+
 		if peer.ExitNode {
-			return "selected but offline"
+			return fmt.Sprintf("selected but offline, %s", lastseen)
```

Comment:
Oh, I just realized, somehow the `last seen` text disappeared. Did you mean to do that? If not, can you add it back in?

You will want to revise the examples in the commit message and PR description, either way you go. 

Thank you!